src/components/Button.tsx

```diff
@@ -0,0 +1,20 @@
+"use client";
+import React from "react";
+
+export interface Button {
+  buttonText: string;
```

Comment:
rename to `children` so it can be used like this `<Button>what ever text I want to go inside</Button>`